dash_docs/chapters/dash_slicer/index.py

```diff
@@ -0,0 +1,135 @@
+import inspect
+
+import dash_html_components as html
+import dash_core_components as dcc
+from dash_docs import styles
+from dash_docs import tools
+from dash_docs import reusable_components as rc
+import dash_slicer
+
+try:
+    from dash_slicer.docs import get_reference_docs
+except ImportError:
+    get_reference_docs = lambda: "TODO"  # while waiting for a new release
+
+
+examples = tools.load_examples(__file__)
+
+
+layout = html.Div([
+    rc.Markdown(f'''
+
+    # Introduction to Dash Slicer
+
+    The ``dash_slicer`` library provides an easy way to visualize 3D image data
+    by slicing along one dimension. Multiple views on the same data can be linked,
+    to help with navigation. There is also support for mask overlays.
+
+    Install Dash Slicer with:
+
+    ```pip install -U dash-slicer```
+
+    The source is on GitHub at [plotly/dash-slicer](https://github.com/plotly/dash-slicer).
+
+    ## Guide
+
+    ### A first example
+
+    Let's get started with a simple example.
+    (The examples on this page are build with dash-slicer version {dash_slicer.__version__})
+    '''),
+
+    rc.Markdown(
+        examples['slicer_example1.py'][0],
+        style=styles.code_container
+    ),
+    html.Div(examples['slicer_example1.py'][1], className='example-container'),
+
+    rc.Markdown('''
+    In the code above, we first load a 3D numpy array (a volumetric image). Then
+    we instantiate a `VolumeSlicer` object with that data. This object is not
+    a Dash component, but has attributes that are. Its `graph` and `slider`
+    are placed in the layout, as well as a handful of `Store` objects that the
```

Comment:
```suggestion
    a Dash component, but has attributes that are. We place its `graph` and
    `slider` in the layout, as well as a handful of `Store` components that the
```
Just to make it clear it's up to you as the user to put them into the layout - otherwise this could be interpreted to mean that by instantiating the `VolumeSlicer` these components are automatically added to the layout.